src/main/java/io/appium/java_client/android/Activity.java

```diff
@@ -0,0 +1,199 @@
+package io.appium.java_client.android;
+
+/**
+ * This is a simple POJO class to support the {@link StartsActivity}.
+ */
+public class Activity {
+    private String appPackage;
+    private String appActivity;
+    private String appWaitPackage;
+    private String appWaitActivity;
+    private String intentAction;
+    private String intentCategory;
+    private String intentFlags;
+    private String optionalIntentArguments;
+    private boolean stopApp;
+
+    /**
+     * A constructor with two mandatory parameters.
+     *
+     * @param appPackage The value for the app package.
+     * @param appActivity The value for the app activity.
+     */
+    public Activity(String appPackage, String appActivity) {
+        this.appPackage = appPackage;
+        this.appActivity = appActivity;
+        this.stopApp = true;
+    }
+
+    /**
+     * Gets the app package value.
+     *
+     * @return The app package value.
+     */
+    public String getAppPackage() {
+        return appPackage;
+    }
+
+    /**
+     * Sets the app package value.
+     *
+     * @param appPackage The app package value.
+     */
+    public void setAppPackage(String appPackage) {
```

Comment:
I'm not sure that it is necessary to keep this method. Maybe it is more senseful to keep only getter for package and make this field final. @SrinivasanTarget What is your opinion?